src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -849,7 +849,13 @@ WORD Terminal::_TakeVirtualKeyFromLastKeyEvent(const WORD scanCode) noexcept
 //      will release this lock when it's destructed.
 [[nodiscard]] std::unique_lock<til::ticket_lock> Terminal::LockForReading()
 {
+#ifdef NDEBUG

```

Comment:
I've always done ifdef DBG and done it the other way. @lhecker would know if this is fine.